### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/CMS-485

### CODE REVIEW

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -252,6 +240,12 @@ msgstr ""
 msgid "Related industries"
 msgstr "Relevante Branchen"
 
+#~ msgid "Website URL"
```
Comment (kowalcj0) [inline locale/de/LC_MESSAGES/django.po]: @eadpearce does it have to be commented out?

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -248,6 +236,12 @@ msgstr ""
 msgid "Related industries"
 msgstr "Industries connexes"
 
+#~ msgid "Investment description"
```
Comment (kowalcj0) [inline locale/fr/LC_MESSAGES/django.po]: @eadpearce same thing here

File: locale/ja/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -246,6 +234,9 @@ msgstr ""
 msgid "Related industries"
 msgstr "関連産業"
 
+#~ msgid "Website URL"
```
Comment (kowalcj0) [inline locale/ja/LC_MESSAGES/django.po]: and here

File: locale/pt/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -249,6 +237,9 @@ msgstr ""
 msgid "Related industries"
 msgstr "Indústrias relacionadas"
 
+#~ msgid "Website URL"
```
Comment (kowalcj0) [inline locale/pt/LC_MESSAGES/django.po]: ditto

File: locale/zh_Hans/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -243,6 +231,9 @@ msgstr ""
 msgid "Related industries"
 msgstr "相关行业"
 
+#~ msgid "Website URL"
```
Comment (kowalcj0) [inline locale/zh_Hans/LC_MESSAGES/django.po]: ditto

File: locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (eadpearce) [reply locale/de/LC_MESSAGES/django.po]: this was autogenerated but yeah i can just delete it

**Pull request merged**